ContextMenu.Droid/ContextMenuScrollViewRenderer.cs

```diff
@@ -148,7 +148,7 @@ private async void OnScrollToRequestedNew(object sender, ScrollToRequestedEventA
 				await Task.Delay(TimeSpan.FromMilliseconds(1));
 				cycle++;
 
-				if (_isDisposed)
+				if (CanHandle() == false)
```

Comment:
I would check this also before entering while loop

Reply:
Done